docs/extensions.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Extensions
+category: 61e72e3a50a88e001a92ee5d
+---
+
+It is possible to extend the Phylum CLI with external sub-commands.
+
+To install an extension, run the following command:
+
+```sh
+phylum extension add path/to/extension
```

Comment:
The future evolution of this command needs to be considered.

If we are going to eventually have a marketplace, we will want `phylum extension add <ext_name>` to go get it from the marketplace and not from a local directory.

So maybe this should be `phylum extension add --local path/to/extension`?

Reply:
Great point. At this stage, though, I feel like we may want to create a separate issue for that and add it in only once we have a clearer direction on how to handle the marketplace.